docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
+from one guest to another.
 
-+ vm1, vm3 -> vnet1
-+ vm2, vm4 -> vnet2
+As a representation of the guest here we use [LXC](https://linuxcontainers.org), which helps users run multiple isolated Linux system containers. Any other virtualization technologies might be used, however we take LXC since it is easy to create/destroy/configure and simple enough to do this verification.
 
-The command "run-vm-all" starts all four virtual machines at once.
+Install LXC
 
-    # cd /root/vnet-test-kvm/vm
-    # ./run-vm-all
-    *** vm1 ***
-    node-id: 1
-    interface name: if-x5vjmksl
-    mac address: 52:54:00:0d:84:8c
-    *** vm2 ***
-    node-id: 2
-    interface name: if-sve1s6xp
-    mac address: 52:54:00:9a:cc:00
-    *** vm3 ***
-    node-id: 3
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+```
+# yum -y install lxc lxc-templates
+```
 
-### Check the IP address of virtual machines
+Create and mount cgroup
 
-You can log into the virtual machines using the telnet command.
+```
+# mkdir /cgroup
+# echo "cgroup /cgroup cgroup defaults 0 0" >> /etc/fstab
+# mount /cgroup
+```
 
-    # telnet localhost PORT-NO
+Create 2 LXC guests
 
-The port numbers are:
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
+```
 
-+ vm1 : 5001
-+ vm2 : 5002
-+ vm3 : 5003
-+ vm4 : 5004
+Configure interfaces of each guest
 
-The ip addresses are assigned as follows. As you can see, vm1 and vm3, share the same ip address, as do vm2 and vm4.
+```
+# vi /var/lib/lxc/inst1/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst1
+lxc.network.ipv4 = 10.100.0.10
+lxc.network.hwaddr = 10:54:FF:00:00:01
+lxc.rootfs = /var/lib/lxc/inst1/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst1
+lxc.autodev = 0
+```
 
-+ vm1 : 10.102.0.10
-+ vm2 : 10.102.0.10
-+ vm3 : 10.102.0.11
-+ vm4 : 10.102.0.11
+```
+# vi /var/lib/lxc/inst2/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst2
+lxc.network.ipv4 = 10.100.0.11
+lxc.network.hwaddr = 10:54:FF:00:00:02
+lxc.rootfs = /var/lib/lxc/inst2/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst2
+lxc.autodev = 0
+```
 
-Log into the virtual machines and make sure their ip addresses are correct.
+Make sure that the IPv4 address and MAC address are the same as what you specify when you create the interface database records.
 
-**ex) vm1**
+Launch the LXC guests then enslave the tap interfaces to the datapath.
 
-    # telnet localhost 5001 <- vm1
-    Trying 127.0.0.1...
-    Connected to localhost.
-    Escape character is '^]'.
+```
+# lxc-start -d -n inst1
+# lxc-start -d -n inst2
 
-    CentOS release 6.4 (Final)
-    Kernel 2.6.32-358.el6.x86_64 on an x86_64
+# ovs-vsctl add-port br0 inst1
+# ovs-vsctl add-port br0 inst2
```

Comment:
I would add a one liner explaining what the ovs-vsctl commands do.

"Now add the lxc container's network interfaces to Open vSwitch's ports. This is basically the same thing as plugging a cable into a switch."